### DISCUSSION TITLE
How to make my website play well with darkreader?

### DISCUSSION THREAD

**Discussion by orazdow** [darkreader/darkreader#8916] [Q&A]
I have darkreader on almost all the time, although it works on most websites, the ones I develop myself often have a single dark div on half the page, or some elements occluded. Is there any guidance on what a site should have in terms of styling to comply with darkreader? I'm not sure what the determining factor is.

**Comment by Gusted** [comment]
I don't think sites should adopt to Dark Reader, but instead Dark Reader being smarter and adopt to the site. Could you make a bug report for this?

**Comment by orazdow** [comment]
Hi, an example I had in mind was this splash page, which is pretty simple html, as I'm not a professional front end dev:
www.thesphinxnyc.com/

Any thoughts on that, or do you think I should still post an issue?
Thanks
